docs/user/build-notifications.rst

```diff
@@ -1,278 +1,24 @@
 Build Notifications and Webhooks
 ================================
 
-.. note::
+Read the Docs features a number of build notification mechanisms.
+Build notifications can alert you when your builds fail so you can take immediate action.
 
-   Currently we don't send notifications or trigger webhooks
-   on :doc:`builds from pull requests </pull-requests>`.
+Email notifications:
+  Read the Docs allows you to configure emails that can be sent on failing builds.
+  This makes sure you know when your builds have failed.
 
-Email notifications
--------------------
+Build Status Webhooks:
+  Build notifications can happen via webhooks.
+  This means that we are able to support a wide variety of services that receive notifications.
 
-Read the Docs allows you to configure emails that can be sent on failing builds.
-This makes sure you know when your builds have failed.
+  Slack and Discord are supported through ready-made templates.
 
-Take these steps to enable build notifications using email:
+  Webhooks can be customized through your own template and a variety of variable substitutions.
 
-* Go to :guilabel:`Admin` > :guilabel:`Notifications` in your project.
-* Fill in the **Email** field under the **New Email Notifications** heading
-* Submit the form
+.. seealso::
```

Comment:
This is a good start to a baseline feature landing page. I'd probably link to the Slack & Discord templates, and perhaps have a screenshot or something here?

Reply:
Noted for subsequent updates iteration :+1: